tests/www/apply/test_process.py

```diff
@@ -2394,6 +2418,146 @@ def test_accept_after_cancel(self):
         assert approval.start_at == job_application.hiring_start_at
         assert job_application.state.is_accepted
 
+    def test_accept_iae__criteria_certification_available(self):
+        ######### Case 1: if BRSA is one the diagnosis criteria,
+        ######### birth place and birth country are required.
+        birthdate = datetime.date(1995, 12, 27)
+        job_application = self.create_job_application(
+            eligibility_diagnosis__with_certifiable_criteria=True,
+            job_seeker__birthdate=birthdate,
+        )
+        url_accept = reverse("apply:accept", kwargs={"job_application_id": job_application.pk})
+
+        employer = job_application.to_company.members.first()
+        self.client.force_login(employer)
+
+        response = self.client.get(url_accept)
+        self.assertContains(response, self.BIRTH_COUNTRY_LABEL)
+        self.assertContains(response, self.BIRTH_PLACE_LABEL)
+
+        # CertifiedCriteriaForm
+        # Birth country is mandatory.
+        post_data = self._accept_view_post_data(job_application=job_application)
+        post_data = {
+            "birth_country": "",
+            "birth_place": "",
+        }
+        response, _ = self.accept_job_application(
+            job_application=job_application, post_data=post_data, assert_successful=False
+        )
+
+        # Wrong birth country and birth place.
+        post_data["birth_country"] = "0012345"
+        post_data["birth_place"] = "008765"
+        response, _ = self.accept_job_application(
+            job_application=job_application, post_data=post_data, assert_successful=False
+        )
```

Comment:
J’allais te proposer un test pour ces cas. :bow: :grin: :clap:

```suggestion
        # Wrong birth country and birth place.
        post_data["birth_country"] = "0012345"
        post_data["birth_place"] = "008765"
        response, _ = self.accept_job_application(
            job_application=job_application, post_data=post_data, assert_successful=False
        )
                assert response.context["form_certified_criteria"].errors == {
            "birth_place": ["Sélectionnez un choix valide. Ce choix ne fait pas partie de ceux disponibles."],
            "birth_country": [
                "Sélectionnez un choix valide. Ce choix ne fait pas partie de ceux disponibles.",
                "Le pays de naissance est obligatoire.",
            ],
        }

```

Reply:
Merci de la suggestion ! C'est ajouté.